cmd/kubelet/app/options/options.go

```diff
@@ -528,6 +521,9 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.Int32Var(&c.ContainerLogMaxFiles, "container-log-max-files", c.ContainerLogMaxFiles, "<Warning: Beta feature> Set the maximum number of container log files that can be present for a container. The number must be >= 2. This flag can only be used with --container-runtime=remote.")
 	fs.StringSliceVar(&c.AllowedUnsafeSysctls, "allowed-unsafe-sysctls", c.AllowedUnsafeSysctls, "Comma-separated whitelist of unsafe sysctls or unsafe sysctl patterns (ending in *). Use these at your own risk.")
 
+	// Experimental flags
+	fs.StringVar(&c.SeccompProfileRoot, "seccomp-profile-root", c.SeccompProfileRoot, "<Warning: Alpha feature> Directory path for seccomp profiles.")
+
```

Comment:
@knabben as part of [seccomp GA](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/20190717-seccomp-ga.md#seccomp-root-path-configuration) graduation, this flag will be deprecated on v1.19. It may be worth marking it as deprecated on this PR.

Reply:
@pjbgf Thanks for it pointing in the KEP. Normally when the flag is moved to deprecated it doesn't go to the configuration file at the same time. Let me know if works.

/cc @saschagrunert